Makefile

```diff
@@ -36,7 +36,7 @@ production-requirements: ## Install requirements for production
 	pipenv install
 
 test: clean ## Run tests and generate coverage report
-	SECRET_KEY=fake DATABASE_URL="sqlite://:memory:" coverage run -m pytest --durations=25 -v
+	coverage run -m pytest --durations=25 -v
```

Comment:
From a quick glance, this appears to mean that the test database will switch from sqlite (which would've needed to be spatialite to support gis capabilities) to postgis

Reply:
Correct. The tests now use whatever database is configured via `DATABASE_URL`. This is ideal because you avoid potentially having tests that work for one database backend but not another. As the project grows, you may wish to go back to using SQLite for speed. Just be aware of the tradeoff.